relay.js

```diff
@@ -125,6 +133,19 @@ export function relayConnect(url, onNotice = () => {}, onError = () => {}) {
     connect()
   } catch (err) {}
 
+  async function trySendWithConfirmation(params) {
+        trySend(params)
```

Comment:
there is a possible race condition here: since you call `trySend()` first it might be that the `OK` event will be received before you have the chance of registering `pendingConfirmations[event.id]`.

Reply:
I didn't realize about that, totally agree. I will think about that